src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+
+type Props = {
+  noCompleteTodos: boolean,
+  filterBy: string,
+  setFilterBy: (value: string) => void,
+  clearCompleted: () => void,
+  leftTodos: number,
+};
+
+const navigation = ['All', 'Active', 'Completed'];
```

Comment:
fill the array with [SortType.ALL, ...]

Reply:
fix it

Reply:
do it in this way: Object.values(SortType)